.github/workflows/build_dev_documentation.yml

```diff
@@ -1,15 +1,17 @@
 name: Build dev documentation
 
 on:
+  push:
+    branches:
+      - docs_new_ui-actions-no-self-hosted
```

Comment:
do we need this part? since I don't see equavalent on https://github.com/huggingface/transformers/blob/master/.github/workflows/build_dev_documentation.yml

Reply:
No you're correct, this can be removed.

Reply:
I see, will keep it for now for testing reasons :)